### ISSUE TITLE
Pretrained Model for Classification Task

### ISSUE THREAD

**Issue by kdmsit** [chao1224/GraphMVP#18]
I am trying to finetune the Pre-trained GraphMVP model on some classification tasks. I am not able to find the path or location for the pre-trained weights for GraphMVP. In the shared google drive link, I found the pre-trained weights for regression tasks. Could you please provide the pre-trained weights of GraphMVP for the classification task (Table-1 in paper)

**Comment by chao1224** [comment]
Hi @kdmsit , it should be the `output.zip` file.

Now I just renamed the two as:
- `GraphMVP_simple_features_for_classification.zip`
- `GraphMVP_complate_features_for_regression.zip`

Feel free to check.

**Comment by kdmsit** [comment]
Thanks for the quick reply. There are many versions in the output directory.

3D_hybrid_02_masking/GEOM_3D_nmol50000_nconf5_nupper1000/CL_1_VAE_1/6_51_10_0.1/0.3_EBM_dot_prod_0.1_normalize_l2_detach_target_2_100_0
3D_hybrid_02_masking/GEOM_3D_nmol50000_nconf5_nupper1000/CL_1_VAE_1/6_51_10_0.1/0.15_EBM_dot_prod_0.2_normalize_l2_detach_target_2_100_0
3D_hybrid_03_masking/GEOM_3D_nmol50000_nconf5_nupper1000/CL_1_VAE_1_AM_1/6_51_10_0.1/0.3_EBM_dot_prod_0.05_normalize_l2_detach_target_2_100_0
3D_hybrid_03_masking/GEOM_3D_nmol50000_nconf5_nupper1000/CL_1_VAE_1_AM_1/6_51_10_0.1/0.15_EBM_dot_prod_0.05_normalize_l2_detach_target_2_100_0
3D_hybrid_03_masking/GEOM_3D_nmol50000_nconf5_nupper1000/CL_1_VAE_1_CP_0.1/6_51_10_0.1/0.3_EBM_dot_prod_0.1_normalize_l2_detach_target_1_100_0
3D_hybrid_03_masking/GEOM_3D_nmol50000_nconf5_nupper1000/CL_1_VAE_1_CP_0.1/6_51_10_0.1/0.15_EBM_dot_prod_0.2_normalize_l2_detach_target_2_100_0


Which one to choose for best results?

**Comment by chao1224** [comment]
Hi @kdmsit,

These six models correspond to the following:
- GraphMVP w/ masking ratio=0.3
- GraphMVP w/ masking ratio=0.15
- GraphMVP-G w/ masking ratio=0.3
- GraphMVP-G w/ masking ratio=0.15
- GraphMVP-C w/ masking ratio=0.3
- GraphMVP-C w/ masking ratio=0.15

You can find their performance in Table 1&2.

I also want to highlight two things:
- All the GraphMVP checkpoints are pretrained on 50K molecules * 5 conformations/each molecule = 250K conformations.
- The GraphMVP framework is agnostic to the backbone models. In the paper, we comprehensively check using GIN as the 2D backbone and SchNet as the 3D backbone, but more advanced 2D GNNs and 3D GNNs could bring in more benefits.

**Comment by kdmsit** [comment]
Thanks a lot for the clarification.
